jOOR-java-8/src/main/java/org/joor/CompileOptions.java

```diff
@@ -60,5 +61,13 @@ public final CompileOptions options(String... newOptions) {
     public final CompileOptions options(List<String> newOptions) {
         return new CompileOptions(processors, newOptions);
     }
+
+    public final boolean skipLoadedClasses() {
+        return skipLoadedClasses;
+    }
+
+    public final void skipLoadedClasses(boolean skipLoadedClasses) {
+        this.skipLoadedClasses = skipLoadedClasses;
+    }
```

Comment:
The other setters don't mutate the `CompileOptions` class, I'd prefer if this were maintained also for new potential flags...